tests/cpp-tests/Classes/ExtensionsTest/CocoStudioActionTimelineTest/ActionTimelineTestScene.cpp

```diff
@@ -66,11 +66,11 @@ void TestActionTimeline::onEnter()
 {
     ActionTimelineBaseTest::onEnter();
 
-    Node* node = CSLoader::createNode("ActionTimeline/DemoPlayer.csb");
-    ActionTimeline* action = CSLoader::createTimeline("ActionTimeline/DemoPlayer.csb");
+    Data data = FileUtils::getInstance()->getDataFromFile("ActionTimeline/DemoPlayer.csb");
+    Node* node = CSLoader::createNodeWithDataBuffer(data);
+    ActionTimeline* action = CSLoader::createTimelineWithDataBuffer(data, "ActionTimeline/DemoPlayer.csb");
```

Comment:
Let me description how CSLoader::createTimelineWithDataBuffer work first: When been called, it will check if ActionTimeline already been loaded, if it was loaded and in cache, return cached item, if it won't loaded, it will be load from data, and after loaded, the new ActionTimeline will be store in cache. The KEY to distinguish ActionTimeline in cache is the csb file name, we send file name here is provide a key for cache, but not use it to read file.